tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 # note that tox expects interpreters to be found at C:\PythonXY,
 # with XY being python version ("27" or "34") for instance
-envlist = py26, py27, py32, py33, py34, docs
+envlist = py{27,34}-pyqt4, py34-pyqt5, py{26,27,32,33,34}-pyside, docs
+
+
 [testenv]
-deps=pytest 
-    pyside    
+deps=pytest
+    py{26,27,32,33,34}-pyside: pyside
 commands=py.test {posargs}
+setenv=
+    py{26,27,32,33,34}-pyside: PYTEST_QT_API = pyside
+    py{26,27,32,33,34}-pyqt4: PYTEST_QT_API = pyqt4
+    py{26,27,32,33,34}-pyqt5: PYTEST_QT_API = pyqt5
+sitepackages=True
+passenv=QT_QPA_PLATFORM_PLUGIN_PATH
```

Comment:
About `QT_QPA_PLATFORM_PLUGIN_PATH`, I don't set it because it is expected to be installed in system python, not in the virtual environment, and I couldn't figure how to access the system python from `tox`. `qutebrowser` sets it because it copies the files over to the virtual environment, right? Perhaps that's what should be done here as well.